File: modules/08-api-boundaries/reject-kotlin/reject.kt
// DOES NOT COMPILE. `balance` is private to Account, so reading it from outside
// the class is rejected:
//
//   error: cannot access 'var balance: Long': it is private in 'Account'.

class Account {
    private var balance: Long = 0
}

fun main() {
    val a = Account()
    println(a.balance)
}

File: modules/08-api-boundaries/reject-swift/reject.swift
// DOES NOT COMPILE. `balance` is private to Account, so touching it from outside
// the struct is rejected:
//
//   error: 'balance' is inaccessible due to 'private' protection level

struct Account {
    private var balance: Int64 = 0
}

let a = Account()
print(a.balance)
